tests/bdd/README.md

```diff
@@ -0,0 +1,31 @@
+# BDD Tests
+
+The BDD tests are written in Python and make use of the pytest-bdd library.
+
+The tests utilise auto-generated code from the OpenApi spec. To generate the client code run:
+```bash
+./../../scripts/generate-openapi-bindings-bdd.sh
+```
+
+The feature files in the `features` directory define the behaviour expected of the control plane. These behaviours are described using the [Gherkin](https://cucumber.io/docs/gherkin/) syntax.
+
+The feature files can be used to auto-generate the test file. For example running `pytest-bdd generate features/volume/replicas.feature > test_volume_replicas.py`
+generates the `test_volume_replicas.py` test file from the `replicas.feature` file.
+
+**:warning: Note: Running pytest-bdd generate will overwrite any existing files with the same name**
+
+## Running the Tests
+Before running any tests run the `setup.sh` script. This sets up the necessary environment to run the tests:
+```bash
+source ./setup.sh
+```
+
+To run all the tests:
+```bash
+pytest .
+```
+
+To run individual test files:
+```bash
+pytest --tc-file=test_config.ini test_volume_observability.py
```

Comment:
Out of curiosity, can we customize the pytest call further to run a specific test within test_volume_observability? (a bit like the moac tests where you could specify the "should create with...")

Reply:
We can using the `-k` option followed by the test name. I've updated the README to include this as well.